src/js/actions/search.js

```diff
@@ -59,11 +61,30 @@ define(function (require, exports) {
         transfers: [dialog.openDialog, dialog.closeDialog]
     };
 
+    /**
+     * Executes Adobe Stock search by opening a URL with the search term
+     *
+     * @return {Promise}
+     */
+    var executeStockSearch = function (searchTerm) {
```

Comment:
I was also debating that. The reason I included it in `actions/search.js` is because for every other search module, their `execute` option was in an actions module pertaining to the executed task. For example, the `executeMenuCommand()` function is held within `actions/menu.js`. Since searching Adobe Stock didn't have a corresponding actions module, I included it in the `search` module. I know it was a bit of a stretch but it follows suit for the other search modules. Having said that, with the second opinion, I think moving it to `adobesstock.js` makes sense and I'll make the change.